Xamarin.Forms.Platform.UAP/CheckBoxRenderer.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System.ComponentModel;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+
+namespace Xamarin.Forms.Platform.UWP
+{
+	public class CheckBoxRenderer : ViewRenderer<CheckBox, FormsCheckBox>
+	{
+		static Brush _tintDefaultBrush = Color.Blue.ToBrush();
+		bool _isDisposed = false;
+
+		protected virtual FormsCheckBox CreateNativeControl()
+		{
+			return new FormsCheckBox()
+			{
+				Style = Windows.UI.Xaml.Application.Current.Resources["FormsCheckBoxStyle"] as Windows.UI.Xaml.Style,			
+			};
+		}
+
+		protected override void OnElementChanged(ElementChangedEventArgs<CheckBox> e)
+		{
+			base.OnElementChanged(e);
+
+			if (e.NewElement != null)
+			{
+				if (Control == null)
+				{
+					var control = CreateNativeControl();
+					control.Checked += OnNativeChecked;
+					control.Unchecked += OnNativeChecked;
+
+					SetNativeControl(control);
+				}
+
+				UpdateIsChecked();
+				UpdateFlowDirection();
+				UpdateTintColor();
+			}
+		}
+
+		protected override void Dispose(bool disposing)
+		{
+			if (Control != null && !_isDisposed)
```

Comment:
See previous comments on `Dipose` overrides.

Reply:
Oh, and this should be checking for deterministic disposal (`disposing == true`), not just `Control != null`.